diff --git constraints/xilinx/vc707.xdc constraints/xilinx/vc707.xdc
index add3a48..a0a40d4 100644
--- constraints/xilinx/vc707.xdc
+++ constraints/xilinx/vc707.xdc
@@ -168,3 +168,25 @@ create_clock -name pci_refclk -period 10 [get_pins *pci_clk_100mhz_buf/O]
 ## no longer needed?
 create_clock -name pci_extclk -period 10 [get_pins *ep7/pcie_ep/inst/inst/gt_top_i/pipe_wrapper_i/pipe_lane[0].gt_wrapper_i/gtx_channel.gtxe2_channel_i/TXOUTCLK]
 set_clock_groups -name ___clk_groups_generated_0_1_0_0_0 -physically_exclusive -group [get_clocks clk_125mhz] -group [get_clocks clk_250mhz]
+
+
+#Bscan ref clk to 125mhz clk
+set_false_path -from [get_clocks -of_objects [get_pins host_ep7/clkgen_pll/CLKOUT0]] -to [get_clocks bscan_refclk]
+set_false_path -from [get_clocks bscan_refclk] -to [get_clocks -of_objects [get_pins host_ep7/clkgen_pll/CLKOUT0]]
+
+#CDC from auroraIntra clocks to/from clkgen_pll_CLKOUT0 (125mhz system clk)
+set_max_delay -from [get_clocks -of_objects [get_pins host_ep7/clkgen_pll/CLKOUT0]] -to [get_clocks auroraI_user_clk_i] -datapath_only 8.0
+set_max_delay -from [get_clocks auroraI_user_clk_i] -to [get_clocks -of_objects [get_pins host_ep7/clkgen_pll/CLKOUT0]] -datapath_only 8.0
+set_max_delay -from [get_clocks -of_objects [get_pins host_ep7/clkgen_pll/CLKOUT0]] -to [get_clocks auroraI_drp_clk_i] -datapath_only 8.0
+set_max_delay -from [get_clocks auroraI_drp_clk_i] -to [get_clocks -of_objects [get_pins host_ep7/clkgen_pll/CLKOUT0]] -datapath_only 8.0
+set_max_delay -from [get_clocks -of_objects [get_pins host_ep7/clkgen_pll/CLKOUT0]] -to [get_clocks auroraI_init_clk_i] -datapath_only 8.0
+set_max_delay -from [get_clocks auroraI_init_clk_i] -to [get_clocks -of_objects [get_pins host_ep7/clkgen_pll/CLKOUT0]] -datapath_only 8.0
+
+
+#CDC from aurora clk (TS_user_clk_i_all) to/from clkgen_pll_CLKOUT0(125MHz clk)
+set_max_delay -from [get_clocks -of_objects [get_pins host_ep7/clkgen_pll/CLKOUT0]] -to [get_clocks TS_user_clk_i_all] -datapath_only 8.0
+set_max_delay -from [get_clocks TS_user_clk_i_all] -to [get_clocks -of_objects [get_pins host_ep7/clkgen_pll/CLKOUT0]] -datapath_only 8.0
+#CDC from aurora sync clk (TS_sync_clk_i_all) to/from clkgen_pll_CLKOUT0(125MHz clk)
+#set_max_delay -from [get_clocks -of_objects [get_pins host_ep7/clkgen_pll/CLKOUT0]] -to [get_clocks TS_sync_clk_i_all] -datapath_only 8.0
+#set_max_delay -from [get_clocks TS_sync_clk_i_all] -to [get_clocks -of_objects [get_pins host_ep7/clkgen_pll/CLKOUT0]] -datapath_only 8.0
+
